### ISSUE TITLE
[Request] Badges for DeviantArt

### ISSUE THREAD

**Issue by emvaized** [badges/shields#10669]
### 📋 Description

It would be great to add support for [DeviantArt](https://www.deviantart.com) badge. 
Here's an example badge which I used on the Github page of [my project](https://github.com/emvaized/modern_inverted_mouse_cursors): 

[![DeviantArt Views](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fbackend.deviantart.com%2Foembed%3Furl%3Dhttps%3A%2F%2Fwww.deviantart.com%2Femvaized%2Fart%2FModern-Inverted-Mouse-Cursors-for-Windows-10-11-1016675934&query=%24.community.statistics._attributes.views&logo=DeviantArt&logoColor=00e59b&label=views&labelColor=black&color=grey)](https://www.deviantart.com/emvaized/art/Modern-Inverted-Mouse-Cursors-for-Windows-10-11-1016675934)

### 🔗 Data

API is described in the [documentation](https://www.deviantart.com/developers/oembed), and allows to retrieve info about specific post using the following URL: 

`https://backend.deviantart.com/oembed?url=${url of the post}` 

Some JSON queries: 

- `$.community.statistics._attributes.downloads` 
- `$.community.statistics._attributes.views` 
- `$.community.statistics._attributes.favorites`
- `$.community.statistics._attributes.comments`

### 🎤 Motivation

Having such a badge will be great for projects which are hosted on DeviantArt. It allows to showcase the downloads, comments, favorites, etc

**Comment by PyvesB** [comment]
Hello @emvaized 👋🏻 

Shields.io hosts integrations for services which are primarily used by developers or which are widely used by developers.

Does DeviantArt fit that category? As an example, we've rejected requests for other image-hosting services such as Unsplash in the past.

**Comment by emvaized** [comment]
@PyvesB 
Fair enough! I'm only aware of DeviantArt being used by cursor or theme developers (such as in my project) or by some Rainmeter skin projects (such as [MontereyRainmeter](https://github.com/creewick/MontereyRainmeter)). While there might be other implications, I agree that they seem quite rare among developers.

**Comment by PyvesB** [comment]
Okay, in that case I'll close the issue for the time being. If more developer-focused usages are surfaced, we can simply re-open it. Thanks for the idea regardless! :)
